Sources/XCHammer/XcodeTarget.swift

```diff
@@ -531,6 +531,33 @@ public class XcodeTarget: Hashable, Equatable {
                 .map { "-D\($0)" }
         }
 
+        func extractSwiftVersion() -> String? {
+            // First look through the swiftcopts to see if `-swift-version <version>` has been specified
+            if let coptsArray = self.attributes[.swiftc_opts] as? [String] {
+                let versionOpt = coptsArray.filter { $0.hasPrefix("-swift-version") }.first
+                let version = versionOpt.map { $0.split(separator: " ") }?.last.map(String.init)
+                if version != nil {
+                    return version
+                }
+            }
+
+            // Second respect swift language version if explicitly defined
+            // This should be less common and i believe was only supported by old rules_apple rules
+            if let version = self.attributes[.swift_language_version] as? String {
```

Comment:
`swift_language_version` is based on `SwiftInfo.swift_version`

https://github.com/bazelbuild/tulsi/blob/c67fdb919a7120b1a42ff8f21eb6028afe204211/src/TulsiGenerator/Bazel/tulsi/tulsi_aspects.bzl#L772

`SwiftInfo.swift_version` is based on `--swiftcopt`: https://github.com/bazelbuild/rules_swift/blob/7a5b35257aec582d2e8faa7bdade110a2cfaab39/swift/internal/compiling.bzl#L206-L225

So in theory, just looking at `swift_language_version` is all that's needed here. Looking through `swiftcopt` shouldn't break anything, but it appears to be redundant.